test/bin/manage_brew_rpms.sh

```diff
@@ -51,7 +60,14 @@ action_download() {
     # cannot be identified easily when running in a CI job
     for arch in x86_64 aarch64 ; do
         local adir
-        adir="${dir}/${arch}"
+        # shellcheck disable=SC2001
+        if [ -z "${ver_type}" ] ; then
+            adir="${dir}/${arch}"
+        else
+            # remove date and commit id from package name and use it in dir name
+            version=$(echo "${package}" | sed 's/.*microshift-\([^-]*\).*/\1/')
+            adir="${dir}/${version}/${arch}"
```

Comment:
Can we simplify this and just use `${version_type}` instead of `${version}`?

Reply:
I think it's not a good idea. I prefer a directory explicitly pointing to the version:

```
$ ls -1d *
4.16.41
4.17.27
4.18.0~0.nightly_2025_06_18_163729
4.18.0~ec.4
4.18.0~rc.10
4.18.11
```

instead of:

```
$ ls -1d *
ec
nightly
rc
zstream
```

or

```
$ ls -1d *
4.16-zstream
4.17-zstream
4.18-ec
4.18-nightly
4.18-rc
4.18-zstream
```

Reply:
Not sure if we are both in the same page here. I'll try to explain myself better.

For me, the final goal of this PR is to have a database/repository (S3 bucket in this case, which is perfect) from where I can fetch any given MicroShift version (RPMs, images or refs) to deploy a MicroShift cluster and run release regression testing.

So my idea/proposal is to put all those artifacts (RPMs, images and refs) in a S3 bucket directory to point to them when triggering the release regression testing. By using the naming approach, `4.18-zstream` I find it difficult to know beforehand which version is that: `4.18.2`? `4.18.11`?

Reply:
For what you were trying to achieve, the versioning makes sense. I missed that point, so thank you for explaining.

As I commented in another thread, changing release-specific storage of artifacts is not what we do and I'm not sure it's a way to go. 

I would suggest to continue with storing the artifacts under a given release because all the system is written to work like this.

Reply:
I'm thinking about this regarding the images, how to build them pointing to the right parent version for upgrades, and I convinced myself we should follow the current approach. Otherwise, I foresee another complete and different approach for parent images, which I think it's not a good idea at all.

So, I change it to follow current approach.